pyproject.toml

```
@@ -1,3 +1,26 @@
+[tool.poetry]
+name = "rasa-sdk"
+version = "1.6.2b1"
+description = "Open source machine learning framework to automate text- and voice-based conversations: NLU, dialogue management, connect to Slack, Facebook, and more - Create chatbots and voice assistants"
+authors = ["Rasa Technologies GmbH <hi@rasa.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+configargparse = "^1.0"
+coloredlogs = "^10.0"
+sanic = "^19.12.2"
+sanic-cors = "0.10.0b1"
```

what's the reason that we can't do this?
```suggestion
sanic-cors = "^0.10.0b1"
```